lib/routemap_northbound.c

```diff
@@ -409,6 +411,14 @@ lib_route_map_entry_exit_policy_modify(struct nb_cb_modify_args *args)
 			rmi->exitpolicy = RMAP_GOTO;
 			break;
 		}
+
```

Comment:
why doesn't this belong in route_map_upd8_dependency.  This way change of the index->nextrm will always notify upper level protocols instead of just this one spot?  I would also add a bool notify because there are places in routemap.c that also call route_map_upd8_dependency and then will update the upper level protocols and we should not do it twice

Reply:
> why doesn't this belong in route_map_upd8_dependency.

I'm not sure how this is related. `lib_route_map_entry_exit_policy_modify()` does not call `route_map_upd8_dependency()` because no new dependencies are introduced by changing `exitpolicy`. However, notifying the dependencies of the changing route-map is needed. And the patch accommodates this. Can you clarify your question?

> This way change of the index->nextrm will always notify upper level protocols instead of just this one spot?

I'm not sure what you mean. `lib_route_map_entry_exit_policy_modify()` does not process `nextrm` changes. `lib_route_map_entry_call_modify()` processes `nextrm` changes and does call `route_map_upd8_dependency()` to add a new dependancy. Can you clarify your question?

Reply:
Can you revisit this comment?